pwiz_tools/Skyline/Model/Export.cs

```diff
@@ -3504,6 +3508,7 @@ public static string GetTargetedHeader(char fieldSeparator)
                     SchedulingReplicateIndex, SchedulingAlgorithm, false, out var windowRT);
                 if (predictedRT.HasValue)
                 {
+                    predictedRT = Math.Max(predictedRT.Value, windowRT + 0.1);
                     retentionTime = (RetentionTimeRegression.GetRetentionTimeDisplay(predictedRT) ?? 0).ToString(CultureInfo);  // Ret. Time (min)
                     deltaRetentionTime = Math.Round(windowRT, 1).ToString(CultureInfo); // Delta Ret. Time (min)

```

Comment:
Yeah. This "Delta Ret. Time (min)" does sound like it should be 1/2 windowRT, and not windowRT, which is a true "window" extending from predictedRT - windowRT/2 to predictedRT + windowRT/2.